### PR TITLE
ci: add OpenStack cli to jenkins image build script

### PR DESCRIPTION
**Pull request by mikkosest**

### CODE REVIEW

**Comment by nordixinfra** [comment]
Can one of the admins verify this patch?

File: ci/scripts/image_scripts/provision_jumphost_jenkins_base_img.sh [hunk 1]
```diff
@@ -11,5 +11,9 @@ SCRIPTS_DIR="$(dirname "$(readlink -f "${0}")")"
 sudo apt install -y \
   openjdk-8-jre
 
+# Install OpenStack CLI
+sudo apt install -y \
+  python-openstackclient
```
Comment (maelk) [inline ci/scripts/image_scripts/provision_jumphost_jenkins_base_img.sh]: No need for a new call to apt, just add it after line 12

Comment (mikkosest) [reply ci/scripts/image_scripts/provision_jumphost_jenkins_base_img.sh]: Done

File: ci/scripts/image_scripts/provision_jumphost_jenkins_base_img.sh [hunk 2]
```diff
@@ -10,6 +10,7 @@ SCRIPTS_DIR="$(dirname "$(readlink -f "${0}")")"
 # Install required packages.
 sudo apt install -y \
   openjdk-8-jre
```
Comment (maelk) [inline ci/scripts/image_scripts/provision_jumphost_jenkins_base_img.sh]: need \

File: resources/docker/gotest/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
+FROM golang:1.11-alpine3.7
```
Comment (maelk) [inline resources/docker/gotest/Dockerfile]: why is this file here ?

Comment (mikkosest) [reply resources/docker/gotest/Dockerfile]: Removed faulty Dockerfile content
